etcd_bootstrap_script.sh

```diff
@@ -41,16 +41,20 @@ check_and_start_etcd(){
         "New")
               wget "$BACKUP_ENDPOINT/initialization/start?mode=$1$FAIL_BELOW_REVISION_PARAMETER" -S -O - ;;
```

Comment:
Don't we need sleep after here? What if the status post new status for some time?

Reply:
Status "New" is posted only once from backup sidecar, after which it goes into Progressing phase. Hence I haven't added the sleep command for the `initialization/start` call